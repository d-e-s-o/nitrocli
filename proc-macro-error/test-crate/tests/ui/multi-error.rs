extern crate test_crate;
use test_crate::make_fn;

make_fn!(multi1, multi2, _, multi3);

fn main() {}
