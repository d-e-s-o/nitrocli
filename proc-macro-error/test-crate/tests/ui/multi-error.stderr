error: multiple error part: multi1

  = note: help message test
  = help: Option help test
  = note: I see what you did here...

 --> $DIR/multi-error.rs:4:10
  |
4 | make_fn!(multi1, multi2, _, multi3);
  |          ^^^^^^

error: multiple error part: multi2

  = note: help message test
  = help: Option help test
  = note: I see what you did here...

 --> $DIR/multi-error.rs:4:18
  |
4 | make_fn!(multi1, multi2, _, multi3);
  |                  ^^^^^^

error: multiple error part: multi3

  = note: help message test
  = help: Option help test
  = note: I see what you did here...

 --> $DIR/multi-error.rs:4:29
  |
4 | make_fn!(multi1, multi2, _, multi3);
  |                             ^^^^^^
